pkg/interfaces/contracts/vault/IVaultErrors.sol

```diff
@@ -284,6 +285,43 @@ interface IVaultErrors {
      */
     error PoolPauseWindowExpired(address pool);
 
+    /*******************************************************************************
+                            Yield-bearing token buffers
```

Comment:
```suggestion
                            ERC4626 token buffers
```
Sometimes we call yield-bearing, and other times we call ERC4626. As a suggestion, maybe we could adopt ERC4626 everywhere, since yield-bearing is a long name (even for test functions, the names are huge when we adopt yieldBearingPool, we could improve code readability with erc4626Pool)